test.ps1

```diff
@@ -116,8 +144,26 @@ New-Module -Name TestHelpers -ScriptBlock {
         $module = Get-Module -Name Pester -ErrorAction Stop
         . $module $ScriptBlock
     }
+
+    function New-Dictionary ([hashtable]$Hashtable) {
+        $d = new-object "Collections.Generic.Dictionary[string,object]"
+        $Hashtable.GetEnumerator() | ForEach-Object { $d.Add($_.Key, $_.Value) }
+
+        $d
+    }
+
+    function Clear-WhiteSpace ($Text) {
+        "$($Text -replace "(`t|`n|`r)"," " -replace "\s+"," ")".Trim()
+    }
+
+    function New-PSObject ([hashtable]$Property) {
+        New-Object -Type PSObject -Property $Property
+    }
```

Comment:
Yes exactly, will remove. I think I've kept it because the syntax was nicer, but I probably did not realize at that moment I can just use [PSCustomObject]

Reply:
done